### ISSUE TITLE
Show total balance of all accounts

### ISSUE THREAD

**Issue by Transisto** [sparrowwallet/sparrow#438]
I think it could be worrisome for someone who does it's first mix to see it's balance split into 4 new wallets.

If there was a total panel it would be reassuring.

I don't know where it would be best displayed, 
The bottom status section is the only thing that's visible on any account but it's a local privacy risk.

If there is enough other common information to be displayed it could be in a separate right tab page.

It could also be right next to every balance,  but there's not much place there.

**Comment by BTCBellyButton** [comment]
I monitor my Join Market accounts via Sparrow and it would be nice to have a "Summary" tab for the wallet with totals from the different accounts. 
![image](https://user-images.githubusercontent.com/68573637/209578138-043d898a-78a3-4714-9d85-9bfeed8f92f9.png)

**Comment by AndySchroder** [comment]
I think that each wallet should have a total for all accounts summarized, AND Sparrow wallet should show the total for all open wallets and all accounts in those open wallets.

Also, there should be some options to differentiate the total (hot and cold wallets), total watching only (cold storage), total multi-sig, total with private keys or a currently connected hardware wallet (total sendable right now), or total with private keys (hot wallets). This might be best in a special summary tab. Also, it would be nice to allow selecting custom groups to summarize the total of. With all this data on a separate summary tab, it might be really helpful to allow the custom groups to also show the transaction history and historical balance for the custom groups, in addition to the current total.

The purpose here is to allow quick assessment that there are no lost coins, and to understand if your balance is growing or shrinking. Also, it allows a quick assessment of what percentage of funds are secured in cold storage vs being a hot wallet so that you know how safe things are.

These should be optional features that can be disabled if someone desires more privacy from onlookers sharing a workspace.

**Comment by marcocattaneogotam** [comment]
Yes, every major software wallet (i.e. Trezor suite, ledger live, bitbox app...)  has a summary screen with the sum of the total balance of all accounts.

The more advanced you make use of bitcoin, the more accounts you use, the more you need a summary screen.

Thanks

**Comment by craigraw** [comment]
Added in f175139f. The Wallet Summary dialog is available from the View menu:

<img width="1124" alt="Screenshot 2023-08-14 at 15 52 12" src="https://github.com/sparrowwallet/sparrow/assets/862166/48d4c712-c6f5-4142-9457-f396ba59bfd5">
